Sources/hostmgr/helpers/S3.swift

```diff
@@ -79,7 +79,19 @@ struct S3Manager {
         var downloadedBytes = 0
         let totalBytes = try getFileSize(region: region, bucket: bucket, key: key)
 
-        let s3Client = try getS3Client(from: client, for: bucket, in: region)
+        // Estimate the time to download the file under 10 MB/s download speed
+        let totalMB = Int64(
+            Measurement<UnitInformationStorage>(
+                value: Double(totalBytes),
+                unit: .bytes
+            )
+            .converted(to: .megabytes)
+            .value
+        )
+        let timeout = totalMB / 10
```

Comment:
Nitpick. `10` looks like a magic number here (even though there's a comment explaining it above).

What do you think of extracting in a constant, e.g.

```
let estimatedDownloadSpeedInMBPS = 10
```

Reply:
👍 Addressed in https://github.com/Automattic/hostmgr/pull/38/commits/eb5a362399efdc61d237452ec80352806b48d8f1